module-4/backend/week_two.md

```diff
@@ -5,17 +5,29 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is Webpack and why is it useful?
+it is a complier, it allows all of your front end tools to be loaded into production
 2. When do you want to use event delegation?
+when you want to add synchronous(sp?) actions. ie on click, on push etc etc
```

Comment:
you'll need it if the element you want to attach an event listener to isn't on the page on document ready!